<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>action_container</class>
 <widget class="QWidget" name="action_container">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#action_container{
  font-family: &quot;Courier New&quot;;
  background-color: #FFFFFF;
}
QGroupBox {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  background-color: rgba(248, 249, 250, 1);
}
QLineEdit {
  padding: 4px 0;
  border: 1px solid #ced4da;
  border-radius: 8px;
  margin-left: 8px;
  padding-left: 4px;
  background-color: #FFFFFF;
  color:#212529;
}
QPlainTextEdit {
	background-color: #FFFFFF;
  color:#212529;
}
QLabel {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: rgb(90, 93, 97);
}
QRadioButton {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: #212529;
}
QComboBox {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: #212529;
}
QPushButton {
  color: #212529;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="action_container_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QComboBox" name="action_name">
       <item>
        <property name="text">
         <string>Marketplace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Discussion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interaction</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="post_container" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="post_container_layout">
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>8</number>
             </property>
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="rightMargin">
              <number>8</number>
             </property>
             <property name="bottomMargin">
              <number>8</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QRadioButton" name="random_radio">
                 <property name="text">
                  <string>random</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="pid_radio">
                 <property name="text">
                  <string>pid</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="manual_radio">
                 <property name="text">
                  <string>manual</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="details_container_w" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_18">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <property name="spacing">
                <number>8</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="images_container_w" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_13">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_14">
                    <item>
                     <widget class="QLabel" name="images_label">
                      <property name="font">
                       <font>
                        <family>Courier New</family>
                        <pointsize>-1</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">margin: 0;</string>
                      </property>
                      <property name="text">
                       <string>Image paths</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="images_input">
                      <property name="styleSheet">
                       <string notr="true">margin: 0;
padding-left: 4px;</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="images_message">
                      <property name="font">
                       <font>
                        <family>Courier New</family>
                        <pointsize>-1</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">margin: 0;
color: red;</string>
                      </property>
                      <property name="text">
                       <string>Image message</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignmentFlag::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="title_container_w" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <property name="topMargin">
                     <number>12</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="title_label">
                      <property name="font">
                       <font>
                        <family>Courier New</family>
                        <pointsize>-1</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">margin: 0;</string>
                      </property>
                      <property name="text">
                       <string>Title</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="title_input">
                      <property name="styleSheet">
                       <string notr="true">margin: 0;
padding-left: 4px;</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="description_container_w" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <property name="bottomMargin">
                     <number>12</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="description_label">
                      <property name="font">
                       <font>
                        <family>Courier New</family>
                        <pointsize>-1</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">margin: 0;</string>
                      </property>
                      <property name="text">
                       <string>Description</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPlainTextEdit" name="description_input"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="pid_container_w" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <item>
                <widget class="QLabel" name="pid_label">
                 <property name="font">
                  <font>
                   <family>Courier New</family>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">margin: 0;</string>
                 </property>
                 <property name="text">
                  <string>PID</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="pid_input">
                 <property name="styleSheet">
                  <string notr="true">margin: 0;
padding-left: 4px;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="pid_message">
                 <property name="font">
                  <font>
                   <family>Courier New</family>
                   <pointsize>-1</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">margin: 0;
color: red;</string>
                 </property>
                 <property name="text">
                  <string>PID message</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="interaction_container" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>interaction</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delete_action_btn">
       <property name="text">
        <string>Delete action</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
